apps/frontend/src/libs/components/tab-container/libs/types/tab-item.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type ReactNode } from "react";
+
+type TabItem = {
+	element: ReactNode;
+	id: string;
```

Comment:
type TabItemKey = 'user-details' | 'my-reviews';

```suggestion
	id: TabItemKey;
```